### PR TITLE
Feature/clearheartbeatsexample

### PR DESCRIPTION
**Pull request by pjohnston**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Creds file
+creds.cfg
```
Comment (StevePC) [inline .gitignore:2]: can we add in a sample creds file so folks know what to create?

**Review by StevePC — CHANGES_REQUESTED** [review]

File: examples/EventsRouter_clearHeartbeats.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import sys
+import logging
+from pprint import pformat
+
+def buildArgs():
+    parser = argparse.ArgumentParser(description='Clear Heatbeats',
+                        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('-v', dest='loglevel', action='store', type=int,
+                        default=30, help='Set script logging level (DEBUG=10,'
+                        ' INFO=20, WARN=30, *ERROR=40, CRTITICAL=50')
```
Comment (StevePC) [inline examples/EventsRouter_clearHeartbeats.py]: why a * next to Error when Warn is the default?

**Review by StevePC — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (pjohnston) [reply .gitignore:2]: creds.cfg already exists just adding the ignore here.

Comment (pjohnston) [reply .gitignore:2]: Renamed it to creds.cfg.sample

**Review by StevePC — APPROVED** [review]

**Pull request merged**